README.adoc

```diff
@@ -120,13 +122,17 @@ The ShrinkWrap test archive's contents are defined in this block, which is going
+The `testGetPropertiesFromEndpoint` test case is an integration test to test the `inventory` service endpoints. The `@RunAsClient` annotation added in this test case indicates that this test case is to be run on the client side, which means the tests are run against the managed container. The endpoint test case first calls the `http://localhost:9080/{WARNAME}/inventory/systems/{hostname}` endpoint with the `localhost` hostname, to add its system properties to the inventory. The test verifies that the system property for the local and service JVM match. Then, the test method calls the `http://localhost:9080/{WARNAME}/inventory/systems` endpoint. The test checks that the inventory should have one host registered and that should be `localhost`. The test also verifies that the system property stored in the inventory for the local and service JVM match.
 
-Next, refer to the `testInventoryResourceFunctions` test case, which is a functional test case to test the functions in the `InventoryResource` class.
+Next, refer to the `testInventoryResourceFunctions` test case, which is a functional test case for the `InventoryResource` class.
 
-CDI is used to inject an instance of the `InventoryResource` class into this test class. It is then tested by the `testGetPropertiesFromFunction` method, which validates that the system property for the local and service JVM match. This test case calls the `listContents()` method to get all systems that are stored in this inventory and verifies that one system is found and it is `localhost`. The functional test case does not do the action to store any system in the inventory, the one system that's found was from the endpoint test case that was run prior to this test case. The `@InSequence` annotation from Arquillian's JUnit integration guarantees the test sequence. The sequences are important for the two test methods, as the results in the first test impacts the second one.
+CDI is used to inject an instance of the `InventoryResource` class into this test class. It is then tested by the `testGetPropertiesFromFunction` method, which validates that the system property for the local and service JVM match. This test case calls the `listContents()` method to get all systems that are stored in this inventory and verifies that one system is found and it is `localhost`.Notice the functional test case does not store any system in the inventory, the `localhost` system is from the endpoint test case that ran prior to this test case. The `@InSequence` Arquillian annotation guarantees the test sequence. The sequences are important for the two test methods, as the results in the first test impacts the second one.
```

Comment:
Missing a space before the sentence starting with "Notice"

Reply:
Fixed